.github/workflows/make-plugin-arm.yml

```diff
@@ -17,9 +17,9 @@ jobs:
         submodules: 'recursive'
     - name: Setup Zig
       run: |
-        mkdir /zig
-        curl 'https://ziglang.org/download/0.9.1/zig-linux-x86_64-0.9.1.tar.xz' | tar xJ --strip-components=1 --directory=/zig
-        ln -s /zig/zig /usr/bin/zig
+        sudo mkdir /zig
+        curl 'https://ziglang.org/download/0.9.1/zig-linux-x86_64-0.9.1.tar.xz' | sudo tar xJ --strip-components=1 --directory=/zig
+        sudo ln -s /zig/zig /usr/bin/zig
```

Comment:
Thanks for the fix!

Curious does this *have* to happen under root directory? If not, would sudo still be necessary?

Reply:
It's about installing for all users (need root permission) or installing for the current user (don't need permission). I think both of them are OK in CI.

Reply:
ahh right.. If you don't mind, i wonder if you could install it for the current user? (in general i'd want to minimize root access unless strictly necessary). Thanks!

Reply:
Sure.

Reply:
lol do you have any simpler solutions?